src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -1833,6 +1833,39 @@ public static LazyStringValue TableValueToId(JsonOperationContext context, int i
             var ptr = tvr.Read(index, out int size);
             return Slice.From(context.Allocator, ptr, size, ByteStringType.Immutable, out slice);
         }
+
+        public class SetDatabaseChangeVectorCommand : TransactionOperationsMerger.MergedTransactionCommand
+        {
+            private readonly List<string> _ignoredList;
+
+            public SetDatabaseChangeVectorCommand(List<string> ignoredList)
+            {
+                _ignoredList = ignoredList;
+            }
+
+            protected override int ExecuteCmd(DocumentsOperationContext context)
+            {
+                Debug.Assert(_ignoredList.Count > 0);
+                DocumentDatabase.RemoveIgnoredChangeVectors(_ignoredList, ref context.LastDatabaseChangeVector);
+                var global = GetDatabaseChangeVector(context);
+                if (global != null)
+                {
+                    DocumentDatabase.RemoveIgnoredChangeVectors(_ignoredList, ref global);
+                    var tree = context.Transaction.InnerTransaction.ReadTree(GlobalTreeSlice);
```

Comment:
I'm pretty sure we have `SaveDatabseChangeVector`, no? If no, create it, this leaks the storage abstraction